Makefile

```diff
@@ -1,5 +1,5 @@
 test:
-	go test ./...
+	go build && go test ./...
```

Comment:
Why this has been changed?

Reply:
I added the build command because I wanted to build the project before running test and then execute it the pr-check flow. I wanted to run both commands that's why changed it.

Reply:
Tests do not require build. Test should stay as it is.

Usually you want to run tests before building.

If you look at the `build` target it has `test` as a dependency. So what you want to run is a `build` target which will run tests beforehand